corehq/apps/hqwebapp/static/hqwebapp/js/select2_handler_v4.js

```diff
@@ -88,26 +85,32 @@ hqDefine("hqwebapp/js/select2_handler_v4", [
                         data['searchString'] = term;
                         return data;
                     },
-                    results: self.processResults,
-                    500: function () {
```

Comment:
What happens now if there's an error?

Reply:
I was looking into it yesterday could not find anything satisfactory. People are suggesting some trick but that doesn't look very accurate to what we want

Reply:
right now It just keep showing 'Searching...'

Reply:
Found a way around. 
Added the  ajax error handler. Earlier  error is used to look like this.
![image](https://user-images.githubusercontent.com/21310920/46134966-be974100-c261-11e8-9552-4fc2a2494454.png)

Now It will look like this. 
![image](https://user-images.githubusercontent.com/21310920/46136140-86453200-c264-11e8-8ec0-7a52c5e420b5.png)

Reply:
Nice!